spec/phone_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'phone'
+
+describe Phone do
+  subject(:phone) { described_class.new }
+
+  it 'can send a text' do
+    allow(phone).to receive(:send_text)
+    expect(phone).to receive(:send_text)
+      .with('Thank you! Your order was placed and will be delivered before'\
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)